alica_engine/include/engine/model/Plan.h

```diff
@@ -49,6 +50,7 @@ class Plan : public AbstractPlan
     const SynchronisationGrp& getSynchronisations() const { return _synchronisations; }
     const RuntimeCondition* getRuntimeCondition() const { return _runtimeCondition; }
     const PreCondition* getPreCondition() const { return _preCondition; }
+    const std::shared_ptr<BasicPlan> getBasicPlan() const { return _basicPlan; }
```

Comment:
No one needs ownership of a basicPlan. Plan exist far longer than anything that can reference them.